### ISSUE TITLE
_ref undefined

### ISSUE THREAD

**Issue by kyoukhana** [yaminncco/vue-sidebar-menu#171]
I am getting _ref undefined when including the menu any ideas as to why. I am using the Vue 3 version and vue-cli

This is how I am importing the component.

```
//main.js
import { createApp } from 'vue'
import App from "./App.vue"
import '@fortawesome/fontawesome-free/js/all.js';
import VueSidebarMenu from 'vue-sidebar-menu'
import 'vue-sidebar-menu/dist/vue-sidebar-menu.css'
import './main.css'

const app = createApp(App)
app.use(VueSidebarMenu)
app.mount("#app")
```

## Here is the error in the code
```
  default: Object(vue__WEBPACK_IMPORTED_MODULE_0__["withCtx"])(function (_ref) {
      var href = _ref.href,
          navigate = _ref.navigate;
      return [Object(vue__WEBPACK_IMPORTED_MODULE_0__["createVNode"])("a", Object(vue__WEBPACK_IMPORTED_MODULE_0__["mergeProps"])(_ctx.$attrs, {
        href: href,
        onClick: navigate
      }), [Object(vue__WEBPACK_IMPORTED_MODULE_0__["renderSlot"])(_ctx.$slots, "default")], 16
      /* FULL_PROPS */
      , ["href", "onClick"])];
    }),
```

**Comment by yaminncco** [comment]
you need to use with vue-router, if you don't want to use vue-router then you can make a custom link component with `linkComponentName` prop

**Comment by yaminncco** [comment]
fixed in v5.0.0-alpha.9
